packages/dapi-grpc/src/core/proto/org.dash.platform.dapi.v0.rs

```diff
@@ -762,7 +762,7 @@ pub mod core_server {
             tonic::Status,
         >;
         /// Server streaming response type for the subscribeToBlockHeadersWithChainLocks method.
-        type subscribeToBlockHeadersWithChainLocksStream: futures_core::Stream<
+        type SubscribeToBlockHeadersWithChainLocksStream: futures_core::Stream<
```

Comment:
This is generated code from proto. There is no point to update it since it will be modified when proto will be updated. It should be fixed in generator or proto I guess

Reply:
60e16ffa326e965c4b409f55e58325a22f62c9b3